src/jobs/generation/MetaGenerator.groovy

```diff
@@ -508,7 +510,7 @@ repos.each { repoInfo ->
 
         if (isPRTest) {
             if (isVSTS) {
-                // TODO: VSTS PR trigger
+                Utilities.addVSTSPRTrigger(jobGenerator, "Gen CI(${repoInfo.server}) - ${repoInfo.branch}/${repoInfo.definitionScript}")
```

Comment:
Will then end up being branch specific?  As in, if I make a PR to release/2.0.0 to change the CI definition, is it going to run a generator intended for master?

Reply:
Changed the groovy script to be branch specific for PR. Will add branch support for TFS plugin as well.